ipc/apis.go

```diff
@@ -72,15 +72,14 @@ func ContainerPreRegister(ctx interface{}, data *grpc.ContainerData) error {
 		return err
 	}
 
-	logrus.Infof("Container pre-registration successfully completed for id: %s",
-		data.Id)
+	logrus.Debugf("Container pre-registration done: id = %s", data.Id)
```

Comment:
Please use "completed" instead of "done" here for consistency with below logs (see my other comments).